scripts/prepare-for-oot-release.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+'use strict';
+
+const {applyPackageVersions, publishPackage} = require('./npm-utils');
+const updateTemplatePackage = require('./update-template-package');
+const fs = require('fs');
+const path = require('path');
+const {cat, echo, exit} = require('shelljs');
+const yargs = require('yargs');
+
+/**
+ * Core packages need to be in sync with the version of React Native that we are basing on.
+ */
+const corePackages = [
```

Comment:
let's take them programmatically from `packages/` folder. E2E tests have the logic for that already